SETUP.md

```diff
@@ -376,23 +91,35 @@ Additionally, you must install the [spark-cosmosdb connector](https://docs.datab
 </details>
 
 
-## Setup guide for Docker
 
-A [Dockerfile](tools/docker/Dockerfile) is provided to build images of the repository to simplify setup for different environments. You will need [Docker Engine](https://docs.docker.com/install/) installed on your system.
+## Setup for Experimental 
+<!-- FIXME FIXME 23/04/01 move to experimental. Have not tested -->
```

Comment:
I think this is much better to what we had before. One thing that might be interesting to do is one section for developers, just to install for example:
```
pip install .[gpu,examples]
```
or if they want the latest development in staging:
```
git checkout staging
pip install .[gpu,example]
```

Reply:
Like the idea! could you add it?  You may have the best idea what to include.

Reply:
makes sense, I'll do a new PR